cfme/provisioning.py

```diff
@@ -188,6 +189,9 @@ def select_security_group(sg):
             ('hardware_monitoring', {
                 version.LOWEST: ui.Select('select#hardware__monitoring'),
                 '5.5': AngularSelect('hardware__monitoring')}),
+            ('boot_disk_size', {
+                version.LOWEST: ui.Select('select#hardware__boot_disk_size'),
+                '5.5': AngularSelect('hardware__boot_disk_size')}),
```

Comment:
At this point we don't care about anything less than 5.5 so you can remove the ui.Select reference I think.